src/libraries/System.Security.Cryptography/tests/osslplugins/e_dntest.c

```diff
@@ -0,0 +1,171 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include <string.h>
+
+#define OPENSSL_SUPPRESS_DEPRECATED
+#include <openssl/engine.h>
+
+static const char* g_engineId = "dntest";
+static const char* g_engineName = "DotNet Test ENGINE";
+
+// PKCS#1 format
+static char g_pubRsaKey[140] = {
```

Comment:
Doesn't C support implicitly sized literal arrays?

```suggestion
static char g_pubRsaKey[] = {
```